File: EjerciciosJavaBackend/src/fases/Fase3.java
package fases;


public class Fase3 {

	public static void main(String[] args) {
		
		int year = 1948;
		int yearBirth = 1968;
		boolean leapYear = false;
		
		for(int i=year; i<=yearBirth; i=i+4) {
			System.out.println(i);
			
			if (i == yearBirth) {
				leapYear = true;
				}
			}
		
		System.out.println("");
		
		if(leapYear == true) {
			String leapBirth = "! Bingo!tu año de nacimiento fue bisiesto";
			System.out.println(yearBirth+leapBirth);
			
		}else {
			String noLeapBirth = "No has nacido en un año bisiesto";
			System.out.println(noLeapBirth);
		
		}		
	  }
	}

File: EjercicioVariablesJavascript/Fase3.js
var year = 1948;
var yearBirth = 1968;
var leapYear = false;
var leapBirth = 'Has nacido en un año bisiesto';
var noLeapBirth = 'No has nacido en un año bisiesto';

for (let index = year; index <= yearBirth; index = index + 4) {
  document.write (index + '<br>');

  if (index == yearBirth) {
    leapYear = true;
  }
}

document.write ('');

if (leapYear == true) {
  document.write ('<br>' + leapBirth);
} else {
  document.write (noLeapBirth);
}
